build-debezium.sh

```diff
@@ -32,7 +32,7 @@ build_docker_image () {
     echo "****************************************************************"
     echo "** Building    debezium/${IMAGE_NAME}:${IMAGE_TAG}"
     echo "****************************************************************"
-    docker build -t "debezium/${IMAGE_NAME}:latest" "${IMAGE_PATH}"
+    docker buildx build --platform linux/amd64,linux/arm64 -t "debezium/${IMAGE_NAME}:latest" "${IMAGE_PATH}"
```

Comment:
Without ``--load`` or ``--push`` the build container lives only in local build cache, which means the script wont be able to push it nor will the consecutive builds be able to use the base image. 

Unfortunately using ``--load`` with multiple architectures hits https://github.com/docker/buildx/issues/59


@matthinrichsen-wf  Did you have a chance to actually try building these images? If not could you look into the issues described above please?